### PR TITLE
Added new liquibase extension quickstart

### PR DESCRIPTION
**Pull request by adrianfiedler**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the documentation must not be updated
- [x] links the documentation update pull request (if needed)
- [x] updates or creates the `README.md` file (with build and run instructions)
- [x] For new quickstart, is located in the directory _component-quickstart_
- [x] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by adrianfiedler** [comment]
hi @gsmet,
one pipeline fails with "install jdk 8 and http response 404" and the second one runs forever.
I have no clue how to fix this?

File: liquibase-quickstart/pom.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      <artifactId>quarkus-resteasy-jsonb</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-liquibase</artifactId>
+    </dependency>
```
[the remark this answers is not in the document]
Comment (adrianfiedler) [reply liquibase-quickstart/pom.xml:63]: I moved junit and restassured dependencies to be last in list

File: liquibase-quickstart/pom.xml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-orm</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jsonb</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
```
[the remark this answers is not in the document]
Comment (adrianfiedler) [reply liquibase-quickstart/pom.xml]: I replaced postgresql with h2

**Comment by gsmet** [comment]
Rebased and force pushed. Sorry, totally forgot about it :/

**Comment by adrianfiedler** [comment]
> Rebased and force pushed. Sorry, totally forgot about it :/

No problem ;-) Anything else to do?

**Comment by gsmet** [comment]
@adrianfiedler nope, I will just wait for CI to be green and merge. Thanks!

**Comment by gsmet** [comment]
And finally merged, thanks!

**Pull request merged**